dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java

```diff
@@ -245,14 +257,22 @@ public static void setFieldInternationalization(
         final long languageId = contentTypeInternationalization.getLanguageId();
         final boolean live = contentTypeInternationalization.isLive();
         final User user = APILocator.systemUser();
-        
-        for (final String propertyName : fieldMap.keySet()) {
-            final String key = String.format("%s.%s.%s", contentType.variable(), fieldMap.get("variable"), propertyName);
-            final String i18nValue = APILocator.getLanguageVariableAPI().getLanguageVariable(
-                    key, languageId, user, live, user == null);
-
-            if (!i18nValue.equals(key) && !i18nValue.equals(fieldMap.get(propertyName).toString())) {
-                fieldMap.put(propertyName, i18nValue);
+
+        try {
+            for (final String propertyName : fieldMap.keySet()) {
+                final String key = String.format("%s.%s.%s", contentType.variable(), fieldMap.get("variable"), propertyName);
+                final String i18nValue = APILocator.getLanguageVariableAPI().getLanguageVariable(
+                        key, languageId, user, live, user == null);
+
+                if (!i18nValue.equals(key) && !i18nValue.equals(fieldMap.get(propertyName).toString())) {
+                    fieldMap.put(propertyName, i18nValue);
+                }
+            }
+        } catch (DotRuntimeException e) {
+            if ( ExceptionUtil.causedBy(e, ConnectException.class)) {
```

Comment:
can we write a test for this?

Reply:
done https://github.com/dotCMS/core/pull/18281/commits/fc44d05d8a9e9041ebf7df076ce9de4b45694bc7#diff-174105054591969073d7cc9d3406b5d5R27